src/components/settings-row/settings-row.test.js

```diff
@@ -27,6 +27,21 @@ context("Tests for SettingsRow component", () => {
       }
     );
 
+    it.each([1, 2, 3, 4, 5, 6])(
+      "should check HTML section heading element is correct when headingLevel is %s",
+      (headingLevels) => {
+        CypressMountWithProviders(
+          <SettingsRowComponent
+            headingLevel={headingLevels}
+            title="foo"
+            description="bar"
+          />
+        );
+
+        cy.get(`h${headingLevels}`).should("exist");
```

Comment:
```suggestion
        cy.get(`h${headingLevels}`).contains("foo");
```
Likewise with the jest test, this could provide some extra specificity in the assertion.